plugins/github-pull-requests-board/src/components/Card/CardHeader.test.tsx

```diff
@@ -65,7 +67,7 @@ describe('<CardHeader/>', () => {
   it('finds commit status in PR Card Header', async () => {
     await renderInTestApp(<CardHeader {...props} />);
     expect(screen.getByText('Commit Status:')).toBeInTheDocument();
-    expect(props.status?.commit.statusCheckRollup.state).toBeTruthy();
+    expect(props.status[0].commit.statusCheckRollup.state).toBeTruthy();
```

Comment:
since the array might be empty, better to be extra carefull

```suggestion
    expect(props.status[0]?.commit.statusCheckRollup.state).toBeTruthy();
```